### PR TITLE
What's New 103 (& Experiments update)

### PR DESCRIPTION
**Pull request by mikehoffms**
**Rendered article for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103?branch=pr-en-us-2007

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+<!-- Video recording of feature in action
+.mp4 attachments -->
+
+See also:
+* [Navigate DevTools with assistive technology](../../../accessibility/navigation.md)
+* [Windows High contrast mode](/fluent-ui/web-components/design-system/high-contrast)
+
+
+<!-- ====================================================================== -->
+## Welcome tool redesign
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103.md:140]: @Reezaali - ready for title/subtitle/ draft text.  rendered:
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103?branch=pr-en-us-2007#welcome-tool-redesign

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md [hunk 2]
```diff
@@ -204,68 +193,125 @@ Adds the **CSP Violations** tool, which displays any Content Security Policy (CS
...
+*  This checkbox is not present in Microsoft Edge Canary v105.
 
-*  This checkbox is present in Microsoft Edge Stable v101.
-*  This checkbox is present in Microsoft Edge Canary v104.
+See also:
+* [Application tool, to manage storage](../storage/application-tool.md)
+
+
+<!-- ====================================================================== -->
+## Show option to expose internals in heap snapshots
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/experimental-features/index.md:239]: @captainbrosset  @zoherghadyali - need text for v105 Experiments checkbox "Show option to expose internals in heap snapshots"

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md [hunk 3]
```diff
@@ -332,75 +380,87 @@ For more information about the visual **Font Editor**, see [Edit CSS font styles
...
+<!-- todo: need text -->
+
+*  This checkbox is present in Microsoft Edge Stable v102.
+*  This checkbox is present in Microsoft Edge Canary v105.
+
 
-*  This checkbox is present in Microsoft Edge Stable v101.
-*  This checkbox is present in Microsoft Edge Canary v104.
+<!-- ====================================================================== -->
+## Enable CSS Authoring hints for inactive rules, deprecated properties, etc.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/experimental-features/index.md:446]: @captainbrosset  @zoherghadyali - need text for v105 Experiments checkbox "Enable CSS Authoring hints for inactive rules, deprecated properties, etc."

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md [hunk 4]
```diff
@@ -424,21 +484,32 @@ In the **Elements** tool, in the **Styles** tab, click on any color preview.  Th
...
-*  This checkbox is present in Microsoft Edge Canary v104.
+*  This checkbox is present in Microsoft Edge Stable v102.
+*  This checkbox is present in Microsoft Edge Canary v105.
 
 See also:
 * [Change colors with the Color Picker](../css/reference.md#change-colors-with-the-color-picker) in _CSS features reference_
 
 
+<!-- ====================================================================== -->
+## Group sources into Authored and Deployed trees
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/experimental-features/index.md:495]: @captainbrosset @zoherghadyali - need text for v105 Experiments checkbox "Group sources into Authored and Deployed trees"

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103.md [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+For history, see [Issue 12769: Improve categorization of objects in heap snapshots](https://bugs.chromium.org/p/v8/issues/detail?id=12769).
+
+See also:
+* [Constructor (group) entries in the heap profiler](../../../memory-problems/heap-snapshots.md#constructor-group-entries-in-the-heap-profiler) in _Record heap snapshots using the Memory tool_.
+* [HiddenClasses and DescriptorArrays](https://v8.dev/blog/fast-properties#hiddenclasses-and-descriptorarrays) - Fast properties in V8.
+* [BigInt object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/BigInt).
+
+
+<!-- ====================================================================== -->
+## DevTools extension for Visual Studio Code: Right-click an HTML file to launch, and Application tool
```
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103.md:41]: I think we can bury the lead a little bit here about the VS Code extension in favor of a shorter, clearer title. The title still has the value prop of being able to do these things directly from VS Code

```suggestion
## Launch the browser from an HTML file and use the Application tool within Visual Studio Code
```

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103.md [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+
+
+<!-- ====================================================================== -->
+## Bug fixes for Focus Mode UI
+
+<!-- Title: Bug fixes for Focus Mode UI -->
+<!-- Subtitle: Various bugs impacting DevTools controls and keyboard shortcuts were fixed in the new Focus Mode UI. -->
+
+In the new **Focus Mode** DevTools UI, various bugs affecting controls and keyboard shortcuts for DevTools have been resolved:
+*  Improved behavior of collapsible tabs in the **Elements** tool.
```
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103.md:66]: Can we get more specific here? @hxlnt, is this mostly about the tabs in the Styles pane? Or something else in the Elements tool that's collapsible?

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103.md [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+* [Simplify DevTools using Focus Mode](../../../experimental-features/focus-mode.md)
+
+
+<!-- ====================================================================== -->
+## The 3D View tool responds better in narrow layouts
+
+<!-- Title: 3D View: Now with more responsive design -->
+<!-- Subtitle: The 3D View toolbar now wraps around in a narrower window. -->
+
+As folks use the **3D View** tool more, especially with the addition of Composited Layers (formerly known as **Layers**), we have found opportunities to improve the UI based on your feedback.  We've made the layout of the 3D View toolbar more responsive so that your development isn't hindered even when DevTools is narrow.  Keep the feedback coming as you check out 3D View!
```
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103.md:85]:
```suggestion
In previous versions of Microsoft Edge, the **3D View** tool did not display correctly in narrow layouts. The toolbar options were cut off when the DevTools were narrow. In Microsoft Edge 103, this issue has been fixed. The toolbar options in 3D View now stack as the width of the DevTools window is decreased.
```

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103.md [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+* [Windows high contrast mode](/fluent-ui/web-components/design-system/high-contrast)
+
+
+<!-- ====================================================================== -->
+## Redesigned Welcome tool
+
+<!-- Title: Welcome tool has expanded content and videos -->
+<!-- Subtitle: The redesigned Welcome tool has expanded Learn and What's New sections, including a new video series. -->
+
+In Microsoft Edge 103, the **Welcome** tool has a new design with improved, expanded content.  It's easier than ever to find content and to contact the DevTools team.
```
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103.md:114]:
```suggestion
In Microsoft Edge 103, the **Welcome** tool has a new design with improved, expanded content.  It's easier than ever to find documentation, videos, and information on new features and bug fixes.
```

We don't have a link to the GitHub repo so I don't think it makes sense to say it's easier to contact the DevTools team. Reeza, we should follow up on adding a GitHub repo link to the Welcome tool

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103.md [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+## DevTools extension for Visual Studio Code: Right-click an HTML file to launch, and Application tool
+
+<!-- Title: DevTools for Visual Studio Code: Right-click to launch, and Application tool -->
+<!-- Subtitle: Use the context menu on any HTML document in Visual Studio Code to launch a new instance of the Microsoft Edge browser and DevTools. -->
+
+To make it easier to start a new embedded instance of Microsoft Edge from within Visual Studio Code, when you right-click an HTML file in the **Explorer**, there's now an **Open with Edge** submenu that has the commands **Open Browser** and **Open Browser with DevTools**:
+
+![Right-click an HTML file to open a browser instance and the 'Edge DevTools' tab.](devtools-103-images/edge-devtools-rightclick-html-file.png)
+
+Also, you can now also use the **Application** tool to gain insights into the storage and service worker information of your project, without leaving Visual Studio Code:
```
Comment (codepo8) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103.md:50]: Repeated "also". Proposal to remove the first one.

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103.md [hunk 10]
```diff
@@ -0,0 +1,146 @@
...
+
+In Microsoft Edge 103, the **Welcome** tool has a new design with improved, expanded content.  It's easier than ever to find documentation, videos, and information about new features and bug fixes.
+
+*  In the **Learn** section, use the links to quickly jump to documentation for some common tasks in DevTools.
+
+*  In the **What's New** section, find out about the latest changes, new features, and bug fixes.
+
+*  In both sections, learn about existing and new web development features through our new video series, [Videos about web development with Microsoft Edge](../../../../dev-videos/index.md).
+
+![The redesigned Welcome tool.](devtools-103-images/welcome-tool.png)
```
Comment (Reezaali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103.md:115]: Updated screenshot to be inserted here.

**Review by Reezaali — APPROVED** [review]
Left a couple of comments.

**Review by zoherghadyali — APPROVED** [review]

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103.md (hunk 6 above)
Comment (hxlnt) [reply microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103.md:59]: Proposed fix:  Improved behavior of collapsible side-panel tabs such as Styles in the **Elements** tool

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103.md [hunk 11]
```diff
@@ -0,0 +1,146 @@
...
+<!-- Subtitle: Various bugs impacting DevTools controls and keyboard shortcuts were fixed in the new Focus Mode UI. -->
+
+In the new **Focus Mode** DevTools UI, various bugs affecting controls and keyboard shortcuts for DevTools have been resolved:
+*  Improved behavior of collapsible tabs in the **Elements** tool.
+*  Fixed some instances in which the name of an open tool wasn't visible.
+*  Improved reliability of **Quick View** controls.
+*  Improved behavior of opening DevTools by using keyboard shortcuts.
+*  Fixed an issue with navigating to a specific line of code in the **Sources** tool by using keyboard shortcuts.
+*  Restored the `Ctrl`+`Shift`+`F` shortcut to open **Search** in **Quick View**:<!-- todo: macOS keys? -->
+
```
Comment (hxlnt) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103.md:65]: Mac shortcut is Command+Option+F.

**Pull request merged**